tck/README.adoc

```diff
@@ -55,7 +55,7 @@ Scenario: Indexing a list with a string
 <2> This step is used to specify an initialization query by scenarios that require certain patterns to exist in the graph.
 <3> If `Q` uses parameters, this step will specify a two-column table detailing the required parameter names and values. The parameter values are in the same format as the expected results.
 <4> The actual Cypher query, `Q`.
-<5> This step specifies the expected results of executing `Q`, represented in a table format if not empty. The first row contains the column names, and subsequent rows contain values. If `Q` contained an `ORDER BY` clause, the result table will be ordered accordingly.
+<5> This step specifies the expected results of executing `Q`, represented in a table format if not empty. The first row contains the column names, and subsequent rows contain values. However, if `Q` contains an `ORDER BY` clause, then this line should instead be `Then the result should be, in order:`, thus ensuring that the result table will be ordered accordingly.
```

Comment:
This is not absolutely true. There is an edge case here that makes things a bit more complicated. If the query has `ORDER BY` and at least two columns, then if for some row the values in the ordered column were equal, and the values in at least one other of the columns were not, we can't use `in order` checking of the rows, because any order of those rows is okay.